web/public/mgmt/config.yml

```diff
@@ -3152,7 +3152,7 @@ collections:
           options: ["expiration", "renewal"],
         }
       - { label: "License Name", name: "licenseName", widget: hidden, default: "" }
-      - { label: "URL Slug", name: "urlSlug", widget: "write-once-read-only-no-space", required: true }
+      - { label: "URL Slug", name: "urlSlug", widget: "no-space", required: true }
```

Comment:
pebble - So the only reason I can think for this being write-once-read-only is because it's being bastardized into an ID of sorts for webflow sync? Or maybe being used like an ID somewhere else. 

So I would be awesome if you could double check real quick that this isn't the case and there aren't more weird changes to be made. 

Other than that looks fine.

Reply:
confirmed, we are not using this collection in webflow